int32 index
float64 x
float64 y
float64 angle
float64 xg
float64 yg
float64 zg
float64 global_angle
float64 curvature
float64 d_curvature
int32 direction
float64 length
float64 v
float64 a
float64 t

FrenetPoint frenet_info
